### PR TITLE
Helm template update for repository

### LINKED ISSUE #157: Repository: infrastructure integration
### What is the expected enhancement?

Create the `helm` configuration to integrate it in the current deployment pipeline.

Epic: #112

### PR DESCRIPTION
**Pull request by akihikokuroda**
### Summary
Fix #157

Add helm template for repository

### CODE REVIEW

**Review by Tansito — CHANGES_REQUESTED** [review]
Instead of adding templates directly how in the Helm we are managing different projects we decided to follow a [sub-charts architecture](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/). Could you create a sub-chart for `quantum-serverless` and `repository`? It should be something like:
```
$ cd quantumserverless/charts
$ helm create repository
Creating repository
```
Similar for what we have for `jupyter` and `manager`.

**Comment by akihikokuroda** [comment]
@Tansito Thanks for review.  I'll move changes into the sub-chart.

**Review by pacomf — APPROVED** [review]

**Review by Tansito — APPROVED** [review]
Thanks for this @akihikokuroda ! I just proposed a little change but nothing problematic 👍

File: infrastructure/helm/quantumserverless/charts/repository/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: repository
+description: A Helm chart for Kubernetes
```
[the remark this answers is not in the document]
Comment (akihikokuroda) [reply infrastructure/helm/quantumserverless/charts/repository/Chart.yaml]: Thanks! I fix it.

**Pull request merged**